### PR TITLE
port screens from old repo

### PR DESCRIPTION
**Pull request by chiemerie-arum**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,12 +21,16 @@
     "next-auth": "^4.18.8",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-hook-form": "^7.42.1",
+    "react-icons": "^4.7.1",
```
Comment (narender56) [inline package.json:25]: why do use react-icons? it's huge lib

Comment (chiemerie-arum) [reply package.json:25]: What alternative could we use?

Comment (narender56) [reply package.json:25]: Icomoon is best, I'll integrate it

Comment (chiemerie-arum) [reply package.json:25]: Alright. Thanks.

Comment (brandonchadlange) [reply package.json:25]: Its fine for now, but I do agree, would rather use tabler icons or something like that

**Pull request merged**